### ISSUE TITLE
[Feature Request] Timer Card

### ISSUE THREAD

**Issue by magnusoverli** [joBr99/nspanel-lovelace-ui#954]
### FEATURE DESCRIPTION
_A clear and concise description of what the feature should do._
A card that has a simple timer function. Allow the user to set a defined amount of time to count down from.
Should overrule the screensaver while active.

I notice that this function is already posted among the screenshot in this repo, but I cannot find any documentation on it.

![image](https://github.com/joBr99/nspanel-lovelace-ui/assets/3475247/a9f9cf89-01a1-48db-9bea-22501c3ce099)

### ADDITIONAL CONTEXT
_Add any other context about the problem here._
A perfect card for NSPanels located in kitchens. Put the caserole on the stove and set the timer on the screen, perfect combo.
### PANEL / FIRMWARE VERION
_Please add the Panel/Firmware Version you are using (EU, US-L or US-P)_
EU

**Comment by magnusoverli** [comment]
On further research I've found that the screenshot is using a timer entity in HA displayed in a grid card.

The timer integration in HA is likely well suited to lots of uses in HA, but not as a kitchen timer.

Building this function inside the NSpanel Lovelace UI could allow to set the timer duration on the display and not having to use another device to set up a static timer in HA. As a kitchen timer, this is not a viable option.

**Comment by joBr99** [comment]
I have page with predefined timers and you can edit the timer of the page as well. Just touch the numbers

**Comment by magnusoverli** [comment]
> I have page with predefined timers and you can edit the timer of the page as well. Just touch the numbers before you start the timer.

Do you have the gode for that setup? I am unable to find such configuration in the docs…?

**Comment by joBr99** [comment]
Just add the home assistant timer to a card

**Comment by joBr99** [comment]
https://raw.githubusercontent.com/joBr99/nspanel-lovelace-ui/main/docs/img/popupTimer.gif

**Comment by magnusoverli** [comment]
Which card are you using there? That looks exactly like what I wanted! :-)

**Comment by joBr99** [comment]
You just need to add a homeassistant timer to your card.

```
- type: cardGrid
  title: Timer
  key: timer
  entities:
    - entity: timer.nudeln
      icon: "mdi:noodles"
```

**Comment by magnusoverli** [comment]
Great! 👍 

Can we also have the NSpanel notify via buzzer that the timer is fisished?
Can we also have the option to count down from more than 1 hour?

**Comment by joBr99** [comment]
maybe you can do 99 minutes, not sure

to get a notification with the buzzer you need to create an automation in homeassistant that is calling the buzzer command on tasmota

**Comment by magnusoverli** [comment]
Also possible on ESPhome I guess...?

Limited to 59 minutes.

Is it possible to implement a digit for selcting hours also?

**Comment by joBr99** [comment]
yes, same with esphome you just need to call the service for the buzzer there

an additional digit for hours is certainly possible to implement, but atm I don't have the time to do so

**Comment by magnusoverli** [comment]
No problem, you've gotten me further than I suspected was already implemented!

I suggest leaving this feature request open, so that when you (or someone else) has the time and motivation, it is there for the taking. :-)
